app/src/main/java/fr/free/nrw/commons/delete/DeleteTask.java

```diff
@@ -161,16 +161,25 @@ protected void onProgressUpdate (Integer... values){
 
     @Override
     protected void onPostExecute(Boolean result) {
+
+        //link to string resources
+        String title_resource = context.getResources().getString(R.string.nominating_for_Deletion);
```

Comment:
Why uppercase? Please look at the other string and use the same typographical conventions, thanks :-)